mex/model/entities/bibliographic-resource.json

```diff
@@ -0,0 +1,416 @@
+            "pattern": "^(?:\\d{4})$",
+            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
+            "type": "string"
+        },
+        "publisher": {
+            "description": "An entity responsible for making the publication available ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "items": {
+                "$ref": "/schema/entities/organization#/identifier"
+            },
+            "sameAs": "http://purl.org/dc/terms/publisher",
+            "type": "array"
+        },
+        "respositoryURL": {
+            "anyOf": [
+                {
+                    "$ref": "/schema/fields/link"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The handle of the publication in the repository, where the publication is stored."
+        },
+        "section": {
+            "description": "The name of the chapter of the publication, the book section belongs to.",
+            "examples": [
+                "Kapitel 1",
+                "A Section About Public Health",
+                "Chapter XII: The History of Public Health",
+                "12",
+                "A",
+                "B."
+            ],
+            "type": "string"
+        },
+        "subtitle": {
+            "description": "The subtitle of the publication.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "title": {
+            "description": "The full title of the publication.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
+            "sameAs": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfBook": {
+            "description": "The title of the book in which the book section is published.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfSeries": {
+            "description": "The title of the book series, the book belongs to.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "volume": {
+            "description": "The volume of the periodical.",
+            "examples": [
+                "24",
+                "34",
+                "1"
+            ],
+            "type": "integer"
```

Comment:
same as with `issue`: are we **sure** this will stay numerical?

and, regardless: needs `anyOf/null, default=null` treatment